### DISCUSSION TITLE
Running CRC on my local laptop needs minimum 9 GB RAM

### DISCUSSION THREAD

**Discussion by rahulsb** [crc-org/crc#3087] [Q&A] [answered]
Hi,
I'm wondering if I can run CRC on my local laptop with 6 or 7 GB free RAM instead if 9 GB.
Is there any settings or possibility to do so ?

Thanks in Advance.

Regards,
Rahul

**Comment by gbraad** [comment, marked as answer]
Minimum requirements for OpenShift are 9GB. More is recommend for stable use when a large project is deployed.
